Autocoders/Python/src/fprime_ac/generators/templates/component/cpp.tmpl

```diff
@@ -474,7 +474,7 @@ namespace ${namespace} {
       (void) snprintf(
           portName,
           sizeof(portName),
-          "%s_${instance}_OutputPort[%d]",
+          "%s_${instance}_OutputPort[%" PRI_FwIndexType "]",
```

Comment:
Similar comment here and for the rest of the changes. Make sure that in each case, the type of the printf specifier matches the type of the value.